data/wsl/Autounattend_BIOS.xml

```diff
@@ -0,0 +1,363 @@
+﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
As mentioned elsewhere, there might be some clarifications regarding the workflow.